src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -38,11 +38,15 @@ public sealed class PackageItemViewModel : INotifyPropertyChanged, ISelectableIt
         private readonly CancellationTokenSource _cancellationTokenSource;
         private readonly IPackageVulnerabilityService _vulnerabilityService;
 
-        public PackageItemViewModel(INuGetSearchService searchService, IPackageVulnerabilityService vulnerabilityService = default)
+        private readonly PackageModel _packageModel;
+
+        public PackageItemViewModel(INuGetSearchService searchService, IPackageVulnerabilityService vulnerabilityService = default, PackageModel packageModel = null)
```

Comment:
I think we can remove this. Do we even need the IPackageVulnerabilityService in the PackageItemViewModel anymore?
Also, is there a legitimate case for a PackageItemViewModel where its model is null? I think we should remove that default unless there's a good reason that I'm not thinking of to have it.

```suggestion
        public PackageItemViewModel(INuGetSearchService searchService, PackageModel packageModel = null)
```

Reply:
VulnerabilityService is used in the UpdatePackageMaxVulnerabilityAsync and ReloadTransitivePackageMetadataAsync methods, I agree we should remove this, but I think that should be done after the hydration otherwise we get errors.

PackageModel is null here just because that breaks all the tests, I'm finish them, but we could leave it as null if we want to merge before fixing the tests